tests/integration_test/history_services_test.py

```diff
@@ -105,279 +105,279 @@ def test_chat_history_with_user_id_contains_special_character(mock_chat_history)
 
 
 @mock.patch('kairon.history.processor.MongoClient', autospec=True)
-def test_visitor_hit_fallback(mock_mongo):
```

Comment:
lot of endpoint changes have been made to tests. we were required to only change the data format to date range wherever month was required. why so many lines of change?

Reply:
I didn't change the endpoints. I moved a few test cases to the top.

Reply:
why? what was the need?